zone/questmgr.cpp

```diff
@@ -2876,6 +2876,20 @@ const char* QuestManager::getguildnamebyid(int guild_id) {
 		return("");
 }
 
+int QuestManager::getguildidbycharid(uint32 char_id) {
+    if (char_id > 0)
+        return database.GetGuildIDByCharID(char_id);
+    else
+        return 0;
+}
+
+int QuestManager::getgroupidbycharid(uint32 char_id) {
+    if (char_id > 0)
```

Comment:
No need to do an if else here, just return 0 default always at the bottom since its implied and just return out like you are if we return a database value

Formatting: Please use brackets